packages/main/src/Card.js

```diff
@@ -193,6 +197,10 @@ class Card extends UI5Element {
 		return this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD);
 	}
 
+	get ariaCardHeaderRoleDescription() {
+		return this.headerInteractive ? this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_INTERACTIVE_TEXT) + " " + this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD_HEADER) : this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD_HEADER);
```

Comment:
Minor suggestion, Is it possible to add a text entry such as
ARIA_ROLEDESCRIPTION_INTERACTIVE_CARD_HEADER=Interactive card header
instead of
ARIA_ROLEDESCRIPTION_INTERACTIVE_TEXT=Interactive

This way you won`t need to concat both strings
```js
return this.headerInteractive ? 
this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_INTERACTIVE_TEXT) + " " 
+ this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD_HEADER) :
this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD_HEADER);

 vs

return this.headerInteractive ? 
this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_INTERACTIVE_CARD_HEADER)  :
this.i18nBundle.getText(ARIA_ROLEDESCRIPTION_CARD_HEADER);

```

Reply:
Yes, i can added, just wanted to avoid the repetition of "Card Header" in the messagebundle

ARIA_ROLEDESCRIPTION_INTERACTIVE_CARD_HEADER=Interactive Card Header

ARIA_ROLEDESCRIPTION_CARD_HEADER=Card Header

Reply:
There is repetition indeed, but this is more safe because  "interactive card header" message as a whole entry will be translated correctly in all languages. When it's split to "interactive" + "car header", the word order might be incorrect in some languages. So, the repetition is fine.